### ISSUE TITLE
Geo search on Point failed under @RedisHash

### ISSUE THREAD

**Issue by jacker996icu** [redis/redis-om-spring#100]
say i have a User class:
```
@RequiredArgsConstructor(staticName = "of")
@AllArgsConstructor(access = AccessLevel.PROTECTED)
@NoArgsConstructor
@Data
@RedisHash
public class User {

    @Id
    @Indexed
    private String id;

    @GeoIndexed
    @NonNull
    private Point homeLoc;
}
```
then redis-om-spring store it in redis hash just like this:
 ```
id: 01GBT9RZJ406M425P0BMGCQ11B
homeLoc.x: 153.616667
homeLoc.y: -28.716667
```
and the index created just like this:
```
"FT.CREATE" "com.redis.hash.model.UserIdx" 
"ON" "HASH" "PREFIX" "1" "com.redis.hash.model.User:" 
"SCHEMA" "id" "AS" "id" "TAG" "SEPARATOR" "," 
"homeLoc" "AS" "homeLoc" "GEO" 
```
when perform geo search:
```
Iterable<User> findByHomeLocNear(Point point, Distance distance);
```
then redis operation:
```
"FT.SEARCH" "com.redis.hash.model.UserIdx" "@homeLoc:[153.59 -28.71 10.0 mi]"
```
I can't find target this way.

I tried to add another filed `String location`:
```
@RequiredArgsConstructor(staticName = "of")
@AllArgsConstructor(access = AccessLevel.PROTECTED)
@NoArgsConstructor
@Data
@RedisHash
public class User {

    @Id
    @Indexed
    private String id;

    @GeoIndexed
    @NonNull
    private Point homeLoc;

    @GeoIndexed
    @NonNull
    private String location;
}
```
and set value "153.616667, -28.716667"
then perform geo search on this filed
```
Iterable<User> findByLocationNear(Point point, Distance distance);
```
then redis operation:
```
"FT.SEARCH" "com.redis.hash.model.UserIdx" "@location:[153.59 -28.71 10.0 mi]"
```
target was found.

**So, am I missed something important to perform Geo search on Point filed under `@RedisHash` annotation? Or should I convert Point into String manually.**

PS:
I have tried to add Converter to convert Point into String with `@WritingConverter`, but did not work at all.
I debugged code and found that `RedisEnhancedKeyValueAdapter` use default `new RedisCustomConversions()`, so I can't register my  Converter.

version:
```
<dependency>
    <groupId>com.redis.om</groupId>
    <artifactId>redis-om-spring</artifactId>
    <version>0.5.1</version>
</dependency>
```

**Comment by bsbodden** [comment]
This is all working on release `0.6.0` which should be dropping in the next day or so.

**Comment by jacker996icu** [comment]
that's great :)

**Comment by bsbodden** [comment]
Addressed in release 0.6.0
